src/components/shared/NewsletterPopup.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useEffect, useState} from "react";
+import {Button, Icon} from "semantic-ui-react";
+
+export default function NewsletterPopup() {
+    const [hidden, setHidden] = useState(true)
+
+    useEffect(() => {
+        if (!localStorage.getItem("SubscribePopupSeen")){
+            if(localStorage.getItem("returnUser") === "true") {
+                setTimeout(function () {
+                    setHidden(false)
+                }, 30000) // 30 seconds in ms
+            }
+        }
+    }, [])
+
+    const onClickHandle = () => {
+        setHidden(true)
+        localStorage.setItem("SubscribePopupSeen", "true")
```

Comment:
In this case we would only register that the user has seen the subscribe popup if they've closed the popup.

Probably not a huge deal but might be better if we set `SubscribePopupSeen` around line 11 instead (i.e. when the popup actually gets shown) so that we don't bombard users show don't figure out how to close the popup.